docs/Get-HelpPreview.md

```diff
@@ -5,25 +5,25 @@ schema: 2.0.0
 
 # Get-HelpPreview
 ## SYNOPSIS
-Preview the output Get-Help would return from an external help file(s).
+Displays your generated external help as **Get-Help** output.
+
 ## SYNTAX
 
 ```
-Get-HelpPreview -Path <String[]> [<CommonParameters>]
```

Comment:
Syntax should not be edited by hands.
It's write-only and generated by `New-MarkdownHelp` and `Update-MarkdownHelp`.

We include `[<CommonParameters>]` to match existing SYNTAX powershell output
![image](https://cloud.githubusercontent.com/assets/816680/16016072/c1f8139e-314d-11e6-875d-ac07c2510377.png)